web/styles/popovers.css

```diff
@@ -617,21 +617,28 @@ ul.popover-group-menu-member-list {
     .stream-list-top-section {
         display: flex;
         margin-top: 12px;
+        align-items: center;
+        margin-bottom: 5px;
 
-        .stream-search {
+        .stream-search-container {
             margin-left: auto;
-            align-self: center;
+            display: grid;
+            grid-template-rows: auto;
+            grid-template-columns: minmax(0, 1fr) 30px;
+        }
+
+        .stream-search {
+            grid-area: 1 / 1 / 2 / 3;
```

Comment:
It would be really nice to add named areas, so that future modifications to the grid don't cause the numbered area references to break.